apps/trustlab/src/lib/data/blockify/postList.js

```diff
@@ -0,0 +1,35 @@
+import formatDate from "@/trustlab/payload/utils/formatDate";
+import { getPosts } from "@/trustlab/utils/post";
+
+async function postList(block, api, context) {
+  const { blockType, deadlineLabel, linkLabel } = block;
+  const { params } = context;
+  const { slugs } = params;
+  const [page] = slugs;
+
+  const { posts = [] } = await getPosts(api, page, {
+    limit: -1,
```

Comment:
Does this mean we're fetching all the posts even if it's 1000?

Reply:
Will handle in #1210